data/en/1_0205.json

```diff
@@ -14,265 +14,44 @@
         "enabled": true,
         "duration": 900
     },
-    "groups": [{
-        "blocks": [{
-                "type": "Introduction",
-                "id": "introduction",
-                "primary_content": [{
-                    "id": "get-started",
-                    "content": [{
-                        "list": [
-                            "On average it takes 10 minutes to complete this survey once you’ve collected the information.",
-                            "Data should relate to all sites in England, Scotland and Wales.",
-                            "You can provide informed estimates if actual figures aren't available.",
-                            "We will treat your data securely and confidentially."
-                        ]
-                    }]
-                }],
-                "preview_content": {
-                    "id": "preview",
-                    "title": "Information you need",
-                    "content": [{
-                            "description": "The total retail turnover must be less than or equal to the value of food sales, alcohol, confectionery and tobacco sales, clothing and footwear sales,  household goods, and other goods."
-                        },
-                        {
-                            "description": "You can select the dates of the period you are reporting for, if the given dates are not appropriate."
-                        }
-                    ],
-                    "questions": [{
-                        "question": "Total retail turnover",
-                        "content": [{
-                            "description": "Include:",
-                            "list": [
-                                "VAT",
-                                "internet sales",
-                                "retail sales from outlets in Great Britain to customers abroad"
-                            ]
-                        }, {
-                            "description": "Exclude:",
-                            "list": [
-                                "revenue from mobile phone network commission and top-up",
-                                "sales from catering facilities used by customers",
-                                "lottery sales and commission from lottery sales",
-                                "sales of car accessories and motor vehicles",
-                                "NHS receipts",
-                                "automotive fuel"
-                            ]
-                        }]
-                    }, {
-                        "question": "Food sales",
+    "sections": [{
+        "id": "section1",
```

Comment:
I think we need to steer away from just numbering the sections.

Reply:
Yeah I can see that it's unclear. It was just _a lot_ of sections to name.

Reply:
I've gone with changing these instances to "default-section" to make it clearer and to try and avoid the temptation for people to add "section-2" etc.